libretroshare/src/services/p3gxsforums.cc

```diff
@@ -844,3 +1005,6 @@ bool RsGxsForumGroup::canEditPosts(const RsGxsId& id) const
 	        id == mMeta.mAuthorId;
 }
 
+RsGxsForumGroup::~RsGxsForumGroup() = default;
```

Comment:
That's useless IMHO: just add {} after the delcaration in the .h, which wouldn't let the reader think that there's something so special in the declared destructor for it to be postponed to the .cpp

Reply:
It is to avoid the compiler complaining about the `vtable` of that class being emitted in every compilation unit, also `default` and `{}` are treated different, because of C++ specification if the costructor/destructor is omitted or declared default the `vtable` lookup is not necessary improving performances, while even if the method is defined `{}` the compiler won't apply that optimization. So the rationale behind this line that I would prefer not to have to write too, is the less costly thing we can do to avoid the `vtable` being emitted in every compilation unit.